PROTOCOL.md

```diff
@@ -424,6 +429,67 @@ enough to get you started. You can watch the data sent and received from
 the server on a regular connection, or look at the client source code
 for a full list of message types.
 
+**Action requests**
+
+These are how the client sends the player's decisions to the server. Most
+requests can be sent with `|RQID`. `RQID` is a unique number used to
+identify which action the request was intended for and is used to protect
+against race conditions with the cancel button.
+
+If an invalid request is sent, the game will replace the missing or
+erroneous request with the first valid choice, which is usually the first
+usable move, or Struggle if there are no usable moves left. Similarly, if
+a switch was expected but a move was sent instead, the game will switch in
+the first usable Pokemon.
+
+`|/team ORDER`
+
+> Chooses the team order. Numbers not listed are displaced to the back by swapping
+> them with the number that took their place. For example `/team 25` sets the team
+> order from default to 253416.
+
+`|/move NUMBER TARGET`
+
+> Requests to use the selected move `NUMBER` to be used on `TARGET` Pokemon. `NUMBER`
+> starts at 1 and can be up to 12 (though usually 4), where `NUMBER` calls the move
+> indexed at `NUMBER - 1` in the Pokemon's set.
+>
+> `TARGET` is optional and only needs to be specified forsingle target moves in
+> doubles/triples formats. Moves with `TARGET` specify which position they are trying
+> to use the move on as a number wherein the opposing Pokemon are positive integers
+> counting up from `1` starting on the right. Ally Pokemon targets are negative
+> integers counting down from `-1` starting on the left. 
+>
+> If `mega` is added as a final parameter, the Pokemon will Mega Evolve if possible.
+
+`|/switch NUMBER`
+
+> Requests to switch the active Pokemon with the Pokemon at position `NUMBER` on the
+> team. In cases where a Pokemon is KO'd, their replacement is chosen with `/switch`.
+> This should correspond to a non-active, non-fainted Pokemon, which means `NUMBER`
+> should be between 2 and 6.
+
+`|/choose ACTION,ACTION,ACTION`
+
+> For doubles/triples formats, decisions are sent for all team positions in the same
+> line separated by commas. `ACTION` can be any of the following: `move`, `switch`,
+> `shift`, `pass`.
+>
+> `move` and `switch` use the same syntax as their respective commands explained above
+> except without the `/`. In triples `/choose shift` requests to `|swap|` the current
+> outside Pokemon to the middle team position. When a team position is empty from the
+> remaining number of Pokemon on the team being fewer than the number of active team
+> positions for the format, `pass` is used to indicate the empty slot is not preforming
```

Comment:
After this I'll do a pullreq to change all cases of "Pokemon" to "Pokémon" everywhere, but for now I'll leave it "Pokemon" here because that's how it is in all of the other locations in this file.